src/docfx/restore/RestoreMap.cs

```diff
@@ -37,18 +37,24 @@ public string GetGitRestorePath(string url)
 
             string FindLastModifiedGitRepository()
             {
-                var (remote, branch) = GitUtility.GetGitRemoteInfo(url);
+                var (remote, branch) = HrefUtility.SplitGitHref(url);
                 var restoreDir = AppData.GetGitDir(url);
 
                 if (!Directory.Exists(restoreDir))
                 {
                     throw Errors.NeedRestore(url).ToException();
                 }
 
-                return Directory.EnumerateDirectories(restoreDir, "*", SearchOption.TopDirectoryOnly)
-                    .Where(f => f.EndsWith(HrefUtility.EscapeUrlSegment(branch)))
-                    .OrderByDescending(f => new DirectoryInfo(f).LastWriteTimeUtc)
-                    .FirstOrDefault();
+                return (
+                    from path in Directory.EnumerateDirectories(restoreDir, "*", SearchOption.TopDirectoryOnly)
+                    let name = Path.GetFileName(path)
+                    where name.StartsWith(HrefUtility.EscapeUrlSegment(branch) + "-") &&
+
+                        // Adding the HEAD file is the last step in a worktree checkout, use it to
+                        // filter out worktrees that are still in progress.
+                        File.Exists(Path.Combine(restoreDir, ".git/worktrees", name, "HEAD"))
```

Comment:
do we have a doc to reference?

Reply:
there is no doc about it, I digged into list worktree and add worktree implementation

Reply:
Maybe we can comment the related code link to our code for reference